lncrawl/assets/web/script.js

```diff
@@ -1,24 +1,42 @@
-// Handle key events
-window.addEventListener("keyup", function (evt) {
+let keyPressTimer = null;
+let navigationInterval = null;
+
+window.addEventListener("keydown", function (evt) {
   function goToHref(el) {
     if (!el) return;
     const href = el.getAttribute("href");
     if (href === "#") return;
     window.location.href = href;
   }
 
-
-    switch (evt.key) {
-      case "ArrowLeft":
+  // Start navigation on keydown event
+  switch (evt.key) {
+    case "ArrowLeft":
+      keyPressTimer = setTimeout(() => {
         goToHref(document.querySelector("a.prev-button"));
-        break;
-      case "ArrowRight":
+        navigationInterval = setInterval(() => {
+          goToHref(document.querySelector("a.prev-button"));
+        }, 50);
+      }, 10);
```

Comment:
we should add more delay to start auto navigation. like: 500ms

Reply:
ok